packages/code/src/Code/Code.styles.ts

```diff
@@ -112,6 +150,48 @@ export function getCodeWrapperVariantStyle(theme: Theme): string {
   `;
 }
 
+export const expandButtonStyle = css`
+  align-items: center;
+  border: none;
+  /**
+    Code wrapper's border radius is 12px. Matching that creates a very 
+    slight gap between the button and the code wrapper. Decreasing by
+    1px removes gap.
+  */
+  border-radius: 0 0 11px 11px;
+  border-width: 1px 0 0 0;
+  border-style: solid;
+  display: flex;
+  font-family: ${fontFamilies.default};
+  font-size: 13px;
+  gap: ${spacing[100]}px;
+  grid-area: expandButton;
+  justify-content: center;
+  transition: all ${transitionDuration.default}ms ease-in-out;
+  z-index: 2;
```

Comment:
Whats the z-index for?

Reply:
It moves the button above the shadows. You'll see this done throughout the component in other styles. I've added a comment to indicate this.